static/js/reducers/order_receipt_test.js

```diff
@@ -0,0 +1,56 @@
+import configureTestStore from 'redux-asserts';
+import { assert } from 'chai';
+import sinon from 'sinon';
+import moment from 'moment';
+
+import { orderReceipt } from './order_receipt';
+import {
+  setInitialTime,
+  setTimeoutActive,
+
+  SET_INITIAL_TIME,
+  SET_TIMEOUT_ACTIVE,
+} from '../actions/order_receipt';
+
+describe('order receipt reducer', () => {
+  let sandbox, store, dispatchThen;
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create();
+    store = configureTestStore(orderReceipt);
+    dispatchThen = store.createDispatchThen();
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+
+    store = null;
+    dispatchThen = null;
+  });
+
+  describe('dialog visibility', () => {
+    [true, false].forEach(bool => {
+      it(`should let you set timeoutActive to ${bool}`, () => {
+        return dispatchThen(setTimeoutActive(bool), [
+          SET_TIMEOUT_ACTIVE
+        ]).then(state => {
+          assert.equal(state.timeoutActive, bool);
+        });
+      });
+    });
```

Comment:
I think you might be able to use `assertReducerResultState` from `reducers/ui_test.js` here, maybe we could move that function out into `reducers/test_utils.js` or something.